.gitlab-ci.yml

```diff
@@ -104,18 +104,17 @@ test-coverage-kcov:
 
 build-linux-ubuntu-amd64:          &build
   stage:                           build
-  only:                            *releaseable_branches
```

Comment:
In my understanding this means the linux binary is in fact build twice for every branch on github. One for testing and one regular version.

Reply:
only:                            &releaseable_branches
    - master
    - stable
    - beta
    - tags

Reply:
This line should be left

Reply:
I'm a bit pedantic here, I want to see build results for all platforms.

Are you saying, the test stage and the build stage are both running linux and therefore duplicate pipelines?